charts/rancher-vsphere-cpi/values.yaml

```diff
@@ -9,12 +9,15 @@ vCenter:
     name: "vsphere-cpi-creds"
     generate: true
 
-# vSphere Tags used to determine the zone and region of a Kubernetes node. This labels will be propagated to NodeLabels
+  # vSphere Tags used to determine the zone and region of a Kubernetes node. This labels will be propagated to NodeLabels
   labels:
     region: "k8s-region"
     zone: "k8s-zone"
     generate: false
 
+  ipFamily: []
+  nodes: {}
```

Comment:
Could you add comments to explain the purpose/usage of those two new options?

Reply:
why would you want to statically configure the node list? I've never seen this as a CPI option.

Reply:
^^^ +1 

HI @bmhughes, could you provide more context?

Reply:
Sure!

The [ipFamily](https://github.com/kubernetes/cloud-provider-vsphere/blob/a096bb88b449efd2bd980c8855393f3cf593b330/docs/book/cloud_config.md#virtualcenter) option sets the families (v4 or v6) for which the CPI will return the node address for, by default this is only IPv4. With [this](https://github.com/cilium/cilium/pull/28953) commit to cilium it will crash if dual stack is enabled without the node having an IPv6 address present.

[nodes](https://github.com/kubernetes/cloud-provider-vsphere/blob/a096bb88b449efd2bd980c8855393f3cf593b330/docs/book/cloud_config.md#nodes) allows influencing the selection of addresses for internal/external address via cidr/vsphere network name/exclude cidr. Not tied to this but something I thought useful to add.

Reply:
The env is required because the dual stack CPI feature is behind a [flag](https://github.com/kubernetes/cloud-provider-vsphere/blob/a096bb88b449efd2bd980c8855393f3cf593b330/pkg/cloudprovider/vsphere/cloud.go#L56)

Reply:
Similarly, for the Nodes part, instead of defining it as a blob {}, could you divide it into 6 individual options? 
Something like this:
```
nodes:
  internalNetworkSubnetCidr: ""
  externalNetworkSubnetCidr: ""
  ..
  .. 
```

That is to not only match the exiting pattern in this file, but also to make it easier to manage.

Reply:
For `ipFamily` the docs are a little misleading at they show the legacy ini config where it was a single string and not the [current yaml config](https://github.com/kubernetes/cloud-provider-vsphere/blob/a096bb88b449efd2bd980c8855393f3cf593b330/pkg/common/config/types_yaml.go#L71) where it is a string array.

I'll make the changes for nodes now.